src/transaction/boot_sr.c

```diff
@@ -2191,7 +2191,14 @@ boot_restart_server (THREAD_ENTRY * thread_p, bool print_restart, const char *db
       boot_Lob_path[0] = '\0';
     }
 
-  /* 
+  // Initialize java stored procedure server
+  error_code = jsp_start_server (db_name, db->pathname);
```

Comment:
Since `jsp_start_server` requires sysprm and error manager, I'd like to propose to move it to 
```c
  /* initialize the type/doain module (also sets up an area) */
  error_code = tp_init ();
  if (error_code != NO_ERROR)
    {
      goto error;
    }

  /* Initialize tsc-timer */
  tsc_init ();
#endif /* !SERVER_MODE */

// HERE!!

  /* *INDENT-OFF* */
#if defined (SA_MODE)
  // thread_manager was not initialized
  assert (thread_p == NULL);
  cubthread::initialize (thread_p);
  assert (thread_p == thread_get_thread_entry_info ());
#endif // SA_MODE
  error_code = cubthread::initialize_thread_entries ();
  if (error_code != NO_ERROR)
    {
      goto error;
    }

```